app/sessionStore.js

```diff
@@ -524,6 +567,20 @@ module.exports.loadAppState = () => {
 }
 
 /**
+ * Called when session is suspected for corruption; this will move it out of the way
+ */
+module.exports.backupSession = () => {
+  const src = getStoragePath()
+  const dest = getTempStoragePath('backup')
+
+  console.log('An error occurred. For support purposes, file "' + src + '" has been renamed to "' + dest + '".')
+
+  fs.rename(src, dest, (err) => {
```

Comment:
this introduces a potential race condition between the rename and the next save. It's unlikely to happen, but I don't think we want to add any more potential race conditions so I think this should be a sync operation and we should use copy instead of rename in case the bug is somehow related to rename